.github/workflows/update-nix-npm.yml

```
@@ -0,0 +1,79 @@
+    - cron: "0 4 * * 1"
+  workflow_dispatch: {}
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-nix-npm:
+    runs-on: ubuntu-latest
+    env:
+      SUMMARY_FILE: ${{ runner.temp }}/summary.txt
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)